lang/python/python-libselinux/Makefile

```diff
@@ -10,25 +10,27 @@ PKG_NAME:=python-$(SRC_NAME)
 PKG_VERSION:=3.1
 PKG_RELEASE:=1
 
-PKG_BUILD_DIR:=$(BUILD_DIR)/$(SRC_NAME)-$(PKG_VERSION)
+PKG_BUILD_DIR:=$(BUILD_DIR)/python-selinux/$(SRC_NAME)-$(PKG_VERSION)
```

Comment:
Overrise PKG_BUILD_DIR to not reuse `PKG_BUILD_DIR` of `libselinux`. This is needed to avoid re-building `libselinux` everytime you want to rebuild `python-libselinux`.

Reply:
Applied in last push.